Charts/base-Django/Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
+- name: postgresql
+  repository: https://charts.bitnami.com/bitnami
+  version: 11.9.13
```

Comment:
maybe use the latest version of postgresql?

Reply:
This field refers to the Helm chart version not the postgresql database version.

Reply:
I think we should remove this file, it does not make sense to have it in the generic base chart .